packages/website/components/contexts/plansContext.js

```diff
@@ -5,7 +5,7 @@ export const sharedPlans = [
     description: 'For those that want to take advantage of more storage',
     price: '$3/mo',
     base_storage: '15GiB',
-    additional_storage: '$0.20 / GiB',
+    additional_storage: '$0.22 / GiB',
```

Comment:
... this implies a change to our stripe fixtures and maybe other backend things. I think it won't matter initially if we dont intend to charge for it, but what the user sees and agrees to does not correspond to how the billing vendor will (eventually) calculate billing.

Reply:
It can be kept at $0.20 / GiB - this was my error.